.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -33,15 +33,16 @@ as any relevant images for UI changes._
 
 ## Added tests?
 
-- [ ] yes
-- [ ] no, because they aren't needed
-- [ ] no, because I need help
+- [ ] Yes
+- [ ] No, and this is why: _please replace this line with details on why tests
```

Comment:
Should this read "replace" though? why not _please describe underneath this line why tests have not been included_

It's minor though 😄

Reply:
@msarit I was following what we do in the [QA section](https://github.com/forem/forem/pull/11088/files#diff-18813c86948efc57e661623d7ba48ff94325c9b5421ec9177f724922dd553a35R31-R32) :) 

The idea being, you should delete that italicized line and add your explanation there